src/services/VerifyAccountRequestProcessor.ts

```diff
@@ -93,108 +97,207 @@ export class VerifyAccountRequestProcessor {
 
   	const name = getFullName(person.name);
   	this.logger.appendKeys({ govuk_signin_journey_id: session.clientSessionId });
-		const timeOfRequest = absoluteTimeNow();
 
-  	let { hmrcUuid } = session;
-  	if (!hmrcUuid) {
-  		hmrcUuid = randomUUID();
-  		await this.BavService.saveHmrcUuid(sessionId, hmrcUuid);
+  	let { vendorUuid } = session;
+
+  	if (!vendorUuid) {
+  		vendorUuid = randomUUID();
+  		await this.BavService.saveVendorUuid(sessionId, vendorUuid);
   	}
 
   	const coreEventFields = buildCoreEventFields(session, this.issuer, clientIpAddress);
-  	await this.BavService.sendToTXMA(
-  		this.txmaQueueUrl,
-  		{
-  			event_name: TxmaEventNames.BAV_COP_REQUEST_SENT,
-  			...coreEventFields,
-  			extensions:{
-  				evidence:[
-						 {
-  						txn: hmrcUuid,
-  					},
-  				],
-			 },
-			 restricted:{
-  				"CoP_request_details": [
-					 {
-  						name,
-  						sortCode,
-  						accountNumber: paddedAccountNumber,
-  						attemptNum: session.attemptCount || 1,
-					 },
-  				],
-		 		},
+
+  	await this.BavService.sendToTXMA(this.txmaQueueUrl, {
+  		event_name: TxmaEventNames.BAV_EXPERIAN_REQUEST_SENT,
+  		...coreEventFields,
+  		extensions: {
+  			evidence: [
+  				{
+  					txn: vendorUuid,
```

Comment:
The txn value should be the third party request identifier. Can we confirm the value for this with Sanka

Reply:
Sure I will check with him tomorrow

Reply:
Waiting for Experian to supply their expRequestId to be used as txn property - shall we just retain this random UUID as a placeholder to avoid this PR being blocked @Stelios-CB